tutorials/continuous-aggs-tutorial.md

```diff
@@ -0,0 +1,239 @@
+
+# Tutorial: Continuous Aggregates
+
+One kind of query that occurs frequently when performing data analysis is
+aggregating data into summaries.
+
+For instance if we want to find out the average of a column for each day in the 
+dataset stored in a hypertable, we would run 
+
+``` sql
+SELECT time_bucket('1 day', timedimesion_column_name), avg(column_name), stddev(column_name)
```

Comment:
nit: can we do `time_bucket('1 day', timedimesion_column_name) as bucket` and then use bucket in the GROUP BY?

Reply:
changed.